ci/install.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -ex
+
+# Install Miniconda
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+
+bash miniconda.sh -b -p $HOME/miniconda
+source $HOME/miniconda/etc/profile.d/conda.sh
+
+# Make new conda environment with required Python version
+conda create -y -n test python=$PYTHON_VERSION
+conda activate test
+
+# Install dependencies
+conda install -c anaconda -y -q requirements/conda.txt
+conda install -y -q requirements/tests.txt
```

Comment:
Sorry, this needs to be

    conda install -c spyder-ide -y -q --file requirements/conda.txt
    conda install -c spyder-ide -y -q --file requirements/tests.txt

Reply:
The packages `entrypoints` and `cookiecutter` are not in spyder-ide channel

Reply:
Yes, they are:

https://anaconda.org/spyder-ide/repo

The thing that was missing from the above commands was the `--file` option, to install the packages in conda.txt and tests.txt.